src/data/PostsData.ts

```diff
@@ -0,0 +1,94 @@
+import { IPost } from "typings/flights"
+
+const PostsData:IPost[] = [
+    {
+        id: 0,
+        thumbnailId: 0,
```

Comment:
Not sure if I remember correctly but on backend this fild is probably nullable, so if there is no thumbnail there will be null here.